### DISCUSSION TITLE
Website hosted on Github Pages looks different than locally hosting it

### DISCUSSION THREAD

**Discussion by imquak** [community/community#67344] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

How the website looks locally:
![imagem](https://github.com/community/community/assets/80677826/65551bcf-ff5a-4d52-adc5-664f65337422)
How it looks when hosted on github:
![imagem](https://github.com/community/community/assets/80677826/81edc032-0154-4e93-97db-5555f9cbcf80)

**Comment by imquak** [comment, marked as answer]
The adblocker was hiding it automatically

**Comment by PaulinusUkoowoUdoh** [comment]
![VIEW ON GITHUB](https://github.com/community/community/assets/144676022/f283efaa-173a-49be-8d75-099af42e3050)
![VIEW ON LIVESERVER](https://github.com/community/community/assets/144676022/65ed29b2-1572-4ee7-8ed6-edcca1161bce)

I am having a problem as well.

**Reply by PaulinusUkoowoUdoh** [reply]
TOP VIEW - Actual on github
BOTTOM - Liveserver on my computer
